src/components/Notification/index.tsx

```diff
@@ -0,0 +1,153 @@
+})<{ ownerState: NotificationProps }>(({ theme, ownerState }) => ({
+	alignItems: 'center',
+	backgroundColor: 'white',
+	borderRadius: shape.border_radius.md.value + 'px',
+	boxShadow: '0px 0px 8px 0px rgba(35, 68, 94, 0.3)', // TODO add a design token for box shadows
+	display: 'flex',
+	gap: theme.spacing(3),
+	maxWidth: 400,
+	padding: theme.spacing(3),
+	position: 'relative',
+	width: '100%',
+}));
+
+const NotificationIllustration = styled('figure', {
+	name: 'Notification',
+	slot: 'illustration',
+})(({ theme }) => ({
+	margin: 0,
+	width: 48,
+	minWidth: 48,
+	[theme.breakpoints.up('sm')]: {
+		width: 72,
+	},
+	'& *': {
+		width: '100%',
+	},
+}));
+
+const NotificationTitle = styled('h3', {
+	name: 'Notification',
+	slot: 'title',
+})(({ theme }) => ({
+	...theme.typography.bodyLg,
+	margin: 0,
+}));
+
+const NotificationContent = styled('div', {
+	name: 'Notification',
+	slot: 'content',
+})(({ theme }) => ({
+	display: 'flex',
+	flexDirection: 'column',
+	flexGrow: 1,
+	gap: theme.spacing(1),
+	[theme.breakpoints.up('sm')]: {
+		gap: theme.spacing(2),
+	},
+}));
+
+const NotificationCloseContainer = styled('nav', {
+	name: 'Notification',
+	slot: 'close',
+})(({ theme }) => ({
+	alignSelf: 'start',
+	[theme.breakpoints.down('sm')]: {
+		backgroundColor: 'white',
+		position: 'absolute',
+		right: theme.spacing(3),
+		top: theme.spacing(3),
+	},
+}));
+
+/**
+ * This component is a notification intended to be displayed at the bottom right
+ * corner of the screen, usually sharing updates on the product with our users.
+ *
+ * This is a Themed Component as documented here: https://mui.com/material-ui/customization/creating-themed-components/.
+ * As such, it can be modified in the theme.
+ */
+export const Notification = React.forwardRef<HTMLDivElement, NotificationProps>(
+	function Notification(inProps, ref) {
+		const props = useThemeProps({ props: inProps, name: 'Notification' });
+		const {
+			illustration,
+			campaignId,
+			children,
+			title,
+			linkHref,
+			linkLabel,
+			onClose,
+			...other
+		} = props;
+		const ownerState = {
+			...props,
+			campaignId,
+			children,
+			illustration,
+			title,
+			linkHref,
+			linkLabel,
+			onClose,
+		};
+		const { state: analytics } = useAnalyticsContext();
+
+		useEffect(() => {
+			analytics.webTracker?.track('Notification: open', { campaignId });
+		}, []);
+
+		return (
+			<NotificationRoot ownerState={ownerState} ref={ref} {...other}>
+				{illustration && (
+					<NotificationIllustration>{illustration}</NotificationIllustration>
+				)}
+				<NotificationContent>
+					<NotificationTitle>{title}</NotificationTitle>
+					{children}
+					<MUILinkWithTracking
+						href={linkHref}
+						eventName="Notification: click on link"
+						eventProperties={{ campaignId }}
+					>
+						{linkLabel}{' '}
+						<FontAwesomeIcon size="sm" icon={faArrowUpRightFromSquare} />
+					</MUILinkWithTracking>
+				</NotificationContent>
+				<NotificationCloseContainer>
```

Comment:
Should this really be taking an entire portion in the right of the Notification? Seems it would be better to have inside the NotificationTitle aligned to the far right (like what we do for the `DialogWithCloseButton`)